include/verilogAST.hpp

```diff
@@ -645,6 +645,19 @@ class ModuleInstantiation : public StructuralStatement {
   ~ModuleInstantiation(){};
 };
 
+class IfDef : public StructuralStatement {
+  public:
+    std::string var;
+    std::vector<std::unique_ptr<StructuralStatement>> body;
```

Comment:
we're just allowing a single statement right now? does this encompass say, multiple statements?